vitrivr-engine-index/src/main/kotlin/org/vitrivr/engine/database/jsonl/retrievable/JsonlRetrievableInitializer.kt

```diff
@@ -2,22 +2,43 @@ package org.vitrivr.engine.database.jsonl.retrievable
 
 import org.vitrivr.engine.core.database.retrievable.RetrievableInitializer
 import org.vitrivr.engine.database.jsonl.JsonlConnection
+import org.vitrivr.engine.database.jsonl.LOGGER
+import java.io.File
+import java.io.IOException
 
 class JsonlRetrievableInitializer(private val connection: JsonlConnection) : RetrievableInitializer {
 
+    private val retrievableFile = File(connection.schemaRoot, "retrievables.jsonl")
```

Comment:
On a fundamental note: (not necessarily something that needs to be changed): I'm always a bit puzzled by why anyone would use the old Java `File` API these days.

Reply:
Because it is so much more convenient, especially with the extensions. But if we are standardizing to the use of Path, I can change it.